pattisdr/osf.io
# Feature/Optimize PageCounter Queries Release Two [ENG-598]

❗️ This is Part Two of Page Counter Migration - pointing against Part One temporarily so changes are more visible.
[Part One of PageCounter Migration here
](https://github.com/CenterForOpenScience/osf.io/pull/8800
)

## Purpose

Part One of the Page Counter Migration adds new fields to the Page Counter migration, migrates the data, and adds some code to populate all fields.

Part Two of the PageCounter migration updates the code to use the new guid/file/action/version fields.



## Changes

- Updates code to query/create new PageCounters using the new guid/file/version/action fields
- Modifies queries on conference view, osfstorage view

## QA Notes
Double check download counts on file detail page, file revisions page, conference submissions page, and file detail in API v2. Double check downloading files increases file downloads as expected.
 
## Side effects




## Ticket

https://openscience.atlassian.net/browse/ENG-598
